src/main/java/com/thebois/views/debug/BeingPathDebugView.java

```diff
@@ -1,35 +1,44 @@
 package com.thebois.views.debug;
 
+import java.util.Collection;
+
 import com.badlogic.gdx.graphics.Color;
 import com.badlogic.gdx.graphics.Texture;
 import com.badlogic.gdx.graphics.g2d.Batch;
 
 import com.thebois.models.Position;
 import com.thebois.models.beings.IBeing;
 import com.thebois.models.beings.IBeingGroup;
+import com.thebois.models.beings.pathfinding.AstarPathFinder;
+import com.thebois.models.beings.pathfinding.IPathFinder;
+import com.thebois.models.world.IWorld;
 import com.thebois.views.game.IView;
 import com.thebois.views.TextureUtils;
 
 /**
- * Displays the paths of all beings.
+ * Recalculates and then displays the paths of all beings.
```

Comment:
Does _the view_ recalculate the paths?

Reply:
To elaborate, now that beings themselves no longer are in charge of or even know about the path that they're taking, this debug view can no longer access it. It's hidden deep within the `MoveAction`, which is entirely inaccessible from the outside (as it should be). 

This means it's basically impossible to render the path they're planning on walking, unless there is some way to get the final destination they're moving towards, and then calculating the path again. Since the beings themselves don't know about this final destination either, the best they can give is their current next-tile-destination. The debug view then calculates the path to that destination and renders it, since it can't get the real path from inside the being.